server/api/reviews_eunji.js

```diff
@@ -0,0 +1,39 @@
+const router = require('express').Router();
+const { Review } = require('../db/models');
+module.exports = router;
+
+// see all reviews
+router.get('/', (req, res, next)=>{
+  Review.findAll()
+    .then( reviews => res.json(reviews))
+    .catch(next)
+})
+// see all reviews by userId
+router.get('/users/:id', (req, res, next)=>{
```

Comment:
Here I wonder whether we can create a route /:userId instead of /users/:id, since api/users seems redundant...